prime-router/src/main/kotlin/fhirengine/translation/hl7/utils/CustomFHIRFunctions.kt

```diff
@@ -102,6 +107,15 @@ object CustomFHIRFunctions {
                 )
             }
 
+            CustomFHIRFunctionNames.ChangeTimezone -> {
+                FHIRPathEngine.IEvaluationContext.FunctionDetails(
+                    "changes the timezome of the date resource to the timezome passed in as the " +
```

Comment:
It says date here which is wrong, but maybe this should say what types it can handle.
Type: time zone

Reply:
Will update the description to say dateTime, and to talk about acceptable types.

As for timezone vs time zone, I'm not seeing anything conclusive, I think in English in general, `time zone` is more common, but http://hl7.org/fhir/R4B/valueset-timezones.html and http://build.fhir.org/valueset-timezones.html both say `timezone`, so I was thinking we should match that. What do you think?